modules/tensor_mechanics/tests/elastic_patch/tests

```diff
@@ -1,7 +1,22 @@
 [Tests]
-  [./elastic_patch]
+  [./test_elastic_patch]
     type = 'Exodiff'
     input = 'elastic_patch.i'
     exodiff = 'elastic_patch_out.e'
   [../]
+
+  [./test_elastic_patch_2Procs]
+    type = Exodiff
+    input = 'elastic_patch.i'
+    exodiff = 'elastic_patch_out.e'
+    min_parallel = '2'
+    prereq = 'test_elastic_patch'
+    platform = 'DARWIN'
```

Comment:
I know. But this is not the same test anymore, so it is worth checking if the restriction is still needed.